packages/landing/static/json-schema/stac-basemaps-extension/1.0/schema.json

```diff
@@ -0,0 +1,139 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://basemaps.linz.govt.nz/json-schema/stac-basemaps-extension/1.0/schema.json#",
+  "title": "LINZ Basemaps Extension",
+  "description": "STAC LINZ Basemaps extension to a SATC item and STAC collection.",
+  "oneOf": [
+    {
+      "allOf": [
+        {
+          "$ref": "https://schemas.stacspec.org/v1.0.0-beta.2/item-spec/json-schema/item.json"
+        },
+        {
+          "$ref": "#/definitions/common"
+        },
+        {
+          "$ref": "#/definitions/item"
+        }
+      ]
+    },
+    {
+      "allOf": [
+        {
+          "$ref": "https://schemas.stacspec.org/v1.0.0-beta.2/collection-spec/json-schema/collection.json"
+        },
+        {
+          "$ref": "#/definitions/common"
+        },
+        {
+          "$ref": "#/definitions/collection"
+        }
+      ]
+    }
+  ],
+  "definitions": {
+    "sampling-method": {
+      "type": "string",
+      "enum": [
+        "nearest",
```

Comment:
do you have any thoughts on how we can keep this upto date?

I wonder if we should just use a typescript -> jsonschema to create this.

Reply:
I don't think the list should change unless the schema version also changes.

Reply:
If I go add another sampling-method to the code and it generates the stac files from that code, this schema is not representative of what is actually being built.

Is this caught anywhere? Like do I know that adding another sampling method to the source will break this schema?

Reply:
I'll make the schema accept any string